src/ol3cesium.js

```diff
@@ -140,6 +140,9 @@ olcs.OLCesium = function(options) {
 
   this.camera_.readFromView();
 
+  // Assures correct canvas size after initialisation
+  this.handleResize_();
```

Comment:
Great catch @gjn.
Please move the block to line 128 (just before the synchronizer loop).
Could you also remove line 141 `this.camera_.readFromView();` which is not necessary?